src/traces/treemap/plot.js

```diff
@@ -84,11 +84,16 @@ function plotOne(gd, cd, element, transitionOpts) {
     var hierarchy = cd0.hierarchy;
     var hasTransition = helpers.hasTransition(transitionOpts);
     var entry = helpers.findEntryWithLevel(hierarchy, trace.level);
+    var isRoot = helpers.isHierarchyRoot(entry);
+
     var maxDepth = helpers.getMaxDepth(trace);
     var hasVisibleDepth = function(pt) {
         return pt.data.depth - entry.data.depth < maxDepth;
     };
 
+    // copy
+    trace._backgroundColor = fullLayout.paper_bgcolor;
```

Comment:
This is a little dirty, but I guess passing `gd` to `styleOne` isn't much better. Passing `paper_bgcolor` as a `styleOne` `opts` would be a pain too ... so let's leave like you have it (for now).

Reply:
Resolving this comment. Please don't address this in this PR :smirk: